### ISSUE TITLE
Optimal Scalable GP Regression Method for 35 Input Variable and around 1 million training points

### ISSUE THREAD

**Issue by Lazloo** [cornellius-gp/gpytorch#521]
I got a little bit lost in the different scalable GP methods. I have a real-life regression problem where I already have applied a Deep-Learning approach for reducing the number of input variable to 35. Further, I have training data set comprising around 1 million data point.

Which Scalable GP approach is best suited in this case?

THX
Lazloo

**Comment by jacobrgardner** [comment]
@Lazloo My recommendation would be to start here with those input sizes: https://github.com/cornellius-gp/gpytorch/blob/master/examples/05_Scalable_GP_Regression_Multidimensional/SVDKL_Regression_SVGP_CUDA.ipynb

This uses VI allowing for stochastic gradient descent with the new `WhitenedVariationalStrategy`, which we've found to be quite stable and perform quite well. In the notebook, we train an SV-DKL model on an example UCI dataset with a few hundred thousand training examples, but because we are training using minibatches it should scale to arbitrarily large datasets, just at the cost of more time (minibatches) per epoch.

**Comment by gpleiss** [comment]
Closing this for now, but @Lazloo feel free to re-open 😄
